### ISSUE TITLE
acc = 50

### ISSUE THREAD

**Issue by shui-tian-ju-shi** [10Ring/LAA-Net#13]
![image](https://github.com/user-attachments/assets/4d2707de-d632-4b53-ab62-0a70527dde5c)
I trained a model by myself. The accuracy (acc) on the validation set is always only 50. When I test with the pre-trained model released by the author, the acc is also only 50. I only changed the relevant parameters in the code. Can you provide some suggestions?

I found through debugging that the mask value for mixing is 0 at the red line in the figure. I don't know why this is.

**Comment by shui-tian-ju-shi** [comment]
I may not have completely downloaded the correct version of the package. Could the problem be related to this?
